src/Microsoft.DocAsCode.HtmlToPdf/ConvertWrapper.cs

```diff
@@ -236,8 +236,8 @@ private ManifestItem FindSiblingCoverPageInManifest(Manifest manifest, ManifestI
         {
             return manifest.Files.SingleOrDefault(f =>
             {
-                return Path.GetDirectoryName(f.SourceRelativePath).StartsWith(Path.GetDirectoryName(tocFile.SourceRelativePath))
-                    && (string.Compare("cover.md", Path.GetFileName(f.SourceRelativePath), true) == 0);
+                return FilePathComparer.OSPlatformSensitiveRelativePathComparer.Equals(Path.GetDirectoryName(f.SourceRelativePath), Path.GetDirectoryName(tocFile.SourceRelativePath))
+                    && (string.Compare("cover.md", Path.GetFileName(f.SourceRelativePath), ignoreCase: true) == 0);
```

Comment:
also use `FilePathComparer.OSPlatformSensitiveRelativePathComparer.Equals`

Reply:
What if I use
`"cover.md".Equals(Path.GetFileName(f.SourceRelativePath), StringComparison.OrdinalIgnoreCase)`?

This is similar to how `toc.yml` is compared in other places:
`"toc.yml".Equals(Path.GetFileName(x.File), StringComparison.OrdinalIgnoreCase)`

Reply:
One corner case is under Linux, the folder may exist both `cover.md` and `COVER.md`. Finding multiple matches will fail `SingleOrDefault`. This is a real corner case, so the current implementation does make sense. Never mind 😄

Reply:
Oh, you're right! I didn't think of that. 😄